src/assets/css/icomoon.css

```diff
@@ -222,12 +222,12 @@
 .icon-database:before {
   content: "\e964";
 }
+.icon-magic:before {
+  content: "\f0d0";
+}
 .icon-link:before {
   content: "\e9cb";
 }
-.icon-magic-wand:before {
-  content: "\f0d0";
-}
 .icon-caret-left:before {
```

Comment:
No idea why `magic-wand` became simply `magic`. Now I can't find much info about the elusive `magic-wand` online...